monitoring-dashboard/components/org.wso2.ei.dashboard.core/src/main/java/org/wso2/ei/dashboard/micro/integrator/delegates/EndpointsDelegate.java

```diff
@@ -43,9 +43,10 @@
     private final DatabaseManager databaseManager = DatabaseManagerFactory.getDbManager();
```

Comment:
Since we are not using DatabaseManager anymore, let's remove this and the respective imports.

Reply:
This is removed.